zeroize/src/lib.rs

```diff
@@ -335,6 +338,22 @@ impl Zeroize for String {
     }
 }
 
+#[cfg(feature = "zeroize_bytes")]
+impl Zeroize for Bytes {
+    fn zeroize(&mut self) {
+        self.clear();
```

Comment:
I do not think it is possible do implement such a function, since it'd have to return a reference to data created in the function.

```rust
pub fn try_as_mut(&mut self) -> Result<&BytesMut, &Bytes> {
    if self.inner.is_mut_safe() {
        Ok(&BytesMut { inner: self.inner })
    } else {
        Err(&self)
    }
}
```

I think I am fine with using `try_mut` and taking ownership. I guess the Zeroize for Bytes could cast it as a pointer and do a volatile write alternatively?